fem/bilinearform.hpp

```diff
@@ -29,6 +29,9 @@ namespace mfem
 /// classes derived from Operator.
 enum class AssemblyLevel
 {
+   /// Legacy fully assembled form, i.e. a global sparse matrix in MFEM,
+   /// Hypre or PETSC format.
+   LEGACYFULL,
```

Comment:
```suggestion
   LEGACYFULL = 0,
```

Reply:
Just a reminder that strongly typed enum don't automatically cast to `int`, and vice-versa.
Meaning that `int assembly = a->GetAssemblyLevel()` will not compile, and `a->SetAssemblyLevel(0)` neither.